articles/_jeux-de-donnees/difference-jeu-de-donnees-et-ressource.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Différences entre un jeu de données et une ressource
+slug: difference-jeu-de-donnees-et-ressource
+---
+# Différences entre un jeu de données et une ressource
```

Comment:
Qu'est-ce qu'un jeu de données et une ressource ?
Le titre actuel implique des similitudes entre les deux et une comparaison ce qui n'est pas le cas.

Reply:
Certes, pourquoi pas, mais enfin il s’agit dans les deux cas de « contenus » publiés sur `data.gouv.fr`, le titre ne me semble donc pas si trompeur que ça. 🙂 

Par ailleurs, je préfère [éviter les titres d’articles à la forme interrogative](https://gds.blog.gov.uk/2013/07/25/faqs-why-we-dont-have-them/), car ils ont tendance à décaler les mots clefs vers la fin de la phrase (ce qui en ralentit la compréhension).